docs/DEVELOPER_GUIDE.md

```diff
@@ -333,5 +333,12 @@ Ideally, the `version` specified should match the release tag of the plugin.
 This helps users and maintainers to easily identify which version of the plugin
 they have installed.
 
+### Automating the releasing of new version
+
+You can use `github actions` to automate the release process. e.g. You can use
+`go-releaser` (or equivalent for other languages) for creation of new release and 
```

Comment:
Don’t mention goreleaser it is not relevant here.

Reply:
done